README.md

```diff
@@ -231,6 +231,33 @@ As it stands, state history is not preserved, but it is still possible to get a
 material.state.previous_values # => ['solid']
 ```
 
+### The "States Getter"
+
+A pluralized, class-level helper method can be used to access all possible state values:
+
+```ruby
+Material.states # => ['solid', 'liquid', 'gas']
```

Comment:
this makes it look like this method returns a list of strings, but is that just the to_s of State?

Reply:
eh i guess it's not really like that since a State _is_ a String. anyway, this makes sense, just forgot how State worked ha